### PR TITLE
Fault-Injection : add/update content based on the revised format

### PR DESCRIPTION
**Pull request by rrsthakur**
*Issue #, if available:*

*Description of changes:* Fault Injection - Add/Update content based on the revised format and fixed an issue with functionality

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: modules/03-network-resiliency/fault-injection/README.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+```
+
+### Test
+
+Test the delay by running a `curl` command against the `catalogdetail` service from within the mesh for user named 'internal' and 'external'.
+
+```sh  
+# Create a multitool pod to be able to use curl from within the mesh
+
+kubectl run multitool --image=praqma/network-multitool -n workshop
```
Comment (iamsouravin) [inline modules/03-network-resiliency/fault-injection/README.md:79]: `praqma/network-multitool` project is moved to [`wbitt/Network-MultiTool`](https://github.com/wbitt/Network-MultiTool).

Change the image to point to `wbitt/network-multitool` instead.

File: modules/03-network-resiliency/fault-injection/README.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  - route:
+    - destination:
+        host: catalogdetail
+        port:
+          number: 3000
+```
+
+### Test
+
+Test the delay by running a `curl` command against the `catalogdetail` service from within the mesh for user named 'internal' and 'external'.
```
Comment (iamsouravin) [inline modules/03-network-resiliency/fault-injection/README.md:181]: This is the abort scenario. Change the start of the sentence to 'Test the abort scenario'.

**Review by iamsouravin — CHANGES_REQUESTED** [review]
Please refer to the review comments in README.md.

**Review by iamsouravin — APPROVED** [review]
LGTM

**Review by praseedasathaye — APPROVED** [review]

**Pull request merged**